src/ann_solo/config.py

```diff
@@ -179,6 +179,10 @@ def __init__(self) -> None:
         self._parser.add_argument(
             '--bin_size', default=0.04, type=float,
             help='ANN vector bin width (default: %(default)s Da)')
+        # ANN vector length after gaussian random projection.
+        self._parser.add_argument(
```

Comment:
**question:** I assume that ultimately the `hash_len` option will then be removed?

Reply:
Correct